services/src/layers/postgres_layer_db.rs

```diff
@@ -917,10 +940,10 @@ where
             SELECT 
                 id, 
                 name,
-                type_name
+                priority
             FROM 
                 layer_providers
-            ORDER BY name ASC
+            ORDER BY priority DESC, name ASC
```

Comment:
I think here and in pro you should also proably do "WHERE priority > -1000", otherwise the user might get fewer than expected results, if they are only filtered out in the handler afterwards.

Reply:
https://github.com/geo-engine/geoengine/pull/920/commits/6cbdadc04008488233e0737326dab2f535e6db30